03 - Showcases & Templates/Plugin Showcases/Breadcrumbs for Comparative Law.md

```diff
@@ -0,0 +1,30 @@
+
+#### Example: Concept notes (Law)
+
+This is specifically about my notes on Law, which I acknowledge is probably very niche but hopefully it can give you some ideas of how to set up your hierarchies and especially how to work with notes in multiple hierarchies. You don't have to choose one!
+
+The first line in my note for Limited Liability Company in Delawarereads:
+
+`#j/US/DEL #t/entity #law/corporate `
+
+This automatically places that note within three different hierarchies: jurisdiction (the "j"), type (the "t"), and topic. These are tags and subtags I already used before the Breadcrumbs plugin and clearly denoted "hierarchies", even if they're not immediately explicit. 
+
+Separating by topic feels the most natural because that's how we're used to categorising things. In school I have subjects like "Corporate Law" and "Constitutional Law", so naturally I want to be able to see all my notes related to this topic easily. Similarly, I have tags and subtags for `#philosophy/scepticism` or `#literature/realism`.
+
+Jurisdiction is also a pretty straightforward hierarchy if you have to deal with multiple jurisdictions like I do. By having the jurisdiction hierarchy, I can easily access all rules that would apply to this note since it's in the same jurisdiction (oversimplifying it a bit, but that's the main purpose). Since it's a different hierarchy, it doesn't care about the topic – whether it's the corporate law of Delaware or the regulatory law of Delaware, it all shows up indiscriminately –, but for me that's *good*, because I want to have an overview of the jurisdiction as a whole to understand how the *system* works. But that's getting into Legal theory.
+
+The hierarchy for types took a bit of figuring out, but for my workflow it is important for the purposes of Comparative Law and more easily navigating different systems. Similarly to before, since it is a different hierarchy altogether, it doesn't care about the topic or the jurisdiction. But the idea is precisely to compare between different types of, in this case, entities in different jurisdictions. How does a LLC in Delaware differ from a LLC in NY? How about from a Sociedade Limitada in Brazil? These are different jurisdictions, but they're the same *type* hierarchy.
+
+Cross-referencing with different topics might sound like it gets messy, but it's eye opening. Another example is look through the `#t/principle` and comparing them. When a judge considers a case they weigh different principles just like this, which is a nice exercise. Or, looking through `#t/authority`, I can consider the American SEC against the Brazilian CVM. 
+
+*And none of these are exclusive.* If I want to focus on one hierarchy, I can fold the others. Breadcrumbs offers me all the potential of hierarchies, and doesn't really take anything away.
```

Comment:
Thank you - This is all really clear and I learned a lot.

I read this page first, and a thing I am not yet understanding, it's how this page relates to the breadcrumbs plug-in.

Reply:
I believe this one is describing the different hierarchies she uses with Breadcrumbs